src/AppInstallerTestExeInstaller/main.cpp

```diff
@@ -1,15 +1,106 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#include <windows.h>
+#include <winreg.h>
+#include <winerror.h>
 #include <iostream>
 #include <fstream>
 #include <filesystem>
 #include <sstream>
 
-// The installer simply prints all args to an output file
+using namespace std::filesystem;
+
+std::wstring registrySubkey = L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\";
+std::wstring defaultProductID = L"{A499DD5E-8DC5-4AD2-911A-BCD0263295E9}";
+
+std::wstring uninstallWideString;
+std::wstring overriddenProductCode;
+
+void GenerateUninstaller() {

```

Comment:
>GenerateUninstaller [](start = 5, length = 19)

Should return string, not use globals.

Reply:
changed method to return string and not use globals